react/features/device-selection/components/VideoInputPreview.js

```diff
@@ -252,4 +262,19 @@ class VideoInputPreview extends Component {
     }
 }
 
-export default translate(VideoInputPreview);
+/**
+ * Maps Redux dispatch to the associated {@code VideoInputPreview}'s props.
+ *
+ * @param {Object} dispatch - Redux dispatch.
+ * @private
+ * @returns {{
+ *     enableFaceTracking: React.PropTypes.func
+ * }}
+ */
+function _mapDispatchToProps(dispatch) {
+    return bindActionCreators({
+        enableFaceTracking
+    }, dispatch);
+}
+
+export default translate(connect(null, _mapDispatchToProps)(VideoInputPreview));
```

Comment:
Instead of using mapDispatchToProps you can directly pass a second parameter { enableFaceTracking }. This will connect the dispatch to props automatically.